subsys/net/l2/ethernet/gptp/gptp_md.c

```diff
@@ -52,6 +52,7 @@ static void gptp_md_follow_up_prepare(struct net_pkt *pkt,
 	hdr->correction_field *= sync_send->rate_ratio;
 	hdr->correction_field += sync_send->follow_up_correction_field;
 	hdr->correction_field <<= 16;
+	hdr->correction_field = htons(hdr->correction_field);
```

Comment:
This is a 64 bit field, shouldn't we do `htonll` here?

Reply:
sorry. fixed.